test/externalTests/bleeps.sh

```diff
@@ -24,6 +24,8 @@ set -e
 source scripts/common.sh
 source test/externalTests/common.sh
 
+REPO_ROOT=$(realpath "$(dirname "$0")/../..")
+
```

Comment:
Is this used?

Reply:
Originally I put it in `common.sh` but IIRC `dirname "$0"` was empty because it's a sourced script. Just moving it here was a quick solution, though I could try harder to make it work in `common.sh`. It must be possible somehow.